src/pages/Explorer/Explorer/Stat.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+    export let comments = 0
+    export let likes = 0
+</script>
+
+<div class="stat row hv-center">
```

Comment:
1. Only vertical alignment is needed, therefore `hv-center` -> `v-center`;
2. Add `c-waterloo`.
3. Add `stats` style rule `stats { fill: var(--waterloo); }`